resources/sync_gateway_configs/bucket_online_offline/bucket_online_offline_default_cc.json

```diff
@@ -11,9 +11,7 @@
     "databases":{
         "db":{
             {{ autoimport }}
-            "unsupported": {
-                {{ xattrs }}
-            },
+            {{ xattrs }}
```

Comment:
In the json, {{ xattrs }} gets replaced with "enable_shared_bucket_access": true,